src/main/java/seedu/address/ui/UpcomingTuitionListPanel.java

```diff
@@ -33,24 +35,20 @@
     public UpcomingTuitionListPanel(ObservableList<Student> studentList) {
         super(FXML);
         ObservableList<Tuition> tuitionList = FXCollections.observableArrayList();
-        ObservableList<Tuition> finalTuitionList = FXCollections.observableArrayList();
 
         populateTuitionList(studentList, tuitionList);
-        filterOneWeekTuitionSessions(tuitionList, finalTuitionList);
-        sortByDate(finalTuitionList);
+        sortByDate(tuitionList);
```

Comment:
might want to consider using `tuitionList.sorted(...)` instead. Might be cleaner, just like how I did it in calendar. but this is fine as well!